### PR TITLE
Add notes for 54

### PR DESCRIPTION
**Pull request by Callisto13**
Everyone check and add things I missed

### CODE REVIEW

**Comment by cPu1** [comment]
If this [bug fix](https://github.com/weaveworks/eksctl/pull/3826) gets reviewed in time, we can include it also.

**Comment by Callisto13** [comment]
Can you run integration tests on that branch now? Then I can start the release quickly after merge

File: docs/release_notes/0.54.0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Improvements
+
+- Include all required policies in example config (#3815)
+- Write kubeconfig to temporary location before `enable flux` runs (#3778)
+
+## Bug Fixes
+
+- Fix panic trying to unlock .kube/eksctl/clusters/<cluster> after locking fails (#3803)
+- Fix panic trying to unlock .kube/config after initial lock failed (#3802)
```
Comment (Callisto13) [inline docs/release_notes/0.54.0.md:16]:
```suggestion
- Fix panic trying to unlock .kube/config after initial lock failed (#3802)
- Handle encrypting OS volume for Bottlerocket (#3820)
```

Commit this once integration tests have passed for it

File: docs/release_notes/0.54.0.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Acknowledgments
+Weaveworks would like to sincerely thank:
+ @wongma7
```
Comment (Callisto13) [inline docs/release_notes/0.54.0.md:20]:
```suggestion
 @wongma7 and @itniels
```

**Review by aclevername — APPROVED** [review]

**Pull request merged**